js/bootstrap-datepicker.js

```diff
@@ -1017,6 +1017,8 @@
 				clsName = this.getClassNames(prevMonth);
 				clsName.push('day');
 
+				var displayDate = prevMonth.getUTCDate();
```

Comment:
Since other tickets like #2037 would like such PR, I think a more abstract name for this option should be use, like `content`. Do you guys agree @acrobat @vsn4ik @hebbet ?